src/foam/nanos/hello/tests.jrl

```diff
@@ -0,0 +1,2 @@
+p({"class":"foam.nanos.hello.HelloTest","id":"HelloTest"})
+p({"class":"foam.nanos.test.Test","id":"HelloTestJS","server":false,"status":2,"code":"var hello = foam.nanos.hello.ClientHelloService.create({\n  delegate: foam.box.HTTPBox.create({\n    url: 'service/hello'\n  }, x)\n}, x);\nhello.hello().then(function(msg) { log(msg); });"})
```

Comment:
Instead of log(msg) the test should probably assert that msg equals the expected value.